### PR TITLE
OHRM-728:Develop my leave screen

### PR DESCRIPTION
**Pull request by RajithaKumara**

### CODE REVIEW

File: src/lib/helpers/leave.ts [hunk 1]
```diff
@@ -18,24 +18,100 @@
...
+  '#0288d9',
+  '#d76900',
+  '#732673',
+  '#405040',
 ];
 
+const LEAVE_STATUS_MAP = {
+  REJECTED: 'Rejected',
+  CANCELLED: 'Cancelled',
+  'PENDING APPROVAL': 'Pending Approval',
```
Comment (samanthajayasinghe) [inline src/lib/helpers/leave.ts:39]: what if const as PENDING_APPROVAL ?

File: src/screens/leave/components/LeaveUsageActions.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+import {StyleSheet, View, ViewProps} from 'react-native';
+import withTheme, {WithTheme} from 'lib/hoc/withTheme';
+import Text from 'components/DefaultText';
+import CardButton from 'screens/leave/components/CardButton';
+import Icon from 'components/DefaultIcon';
+import {MY_LEAVE} from 'screens';
+import {navigate} from 'lib/helpers/navigation';
+
+class LeaveUsageActions extends React.Component<LeaveUsageActionsProps> {
+  calculateProgress = (total?: number, used?: number) => {
```
Comment (samanthajayasinghe) [inline src/screens/leave/components/LeaveUsageActions.tsx]: better to add some js doc blocks

File: src/lib/helpers/leave.ts (hunk 1 above)
Comment (RajithaKumara) [reply src/lib/helpers/leave.ts:39]: Those const value coming from OS instance. Here I only maintain that mapping

File: src/screens/leave/components/LeaveUsageActions.tsx (hunk 2 above)
Comment (RajithaKumara) [reply src/screens/leave/components/LeaveUsageActions.tsx]: Done

**Pull request merged**